src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -272,14 +273,18 @@ hoi4::States CreateStates(const std::map<int, vic3::State>& states,
              final_connected_province_sets.size());
       }
 
+      const int64_t total_manpower = vic3_state_itr->second.GetPopulation();
       for (const auto& province_set: final_connected_province_sets)
       {
+         const int manpower = static_cast<int>(total_manpower * province_set.size() / hoi4_provinces.size());
```

Comment:
There is room for improvement here though. If one vic3 substate has that state's "city" province, that province alone could carry quarter or half the state population, as that quite usually is the RL case.

Reply:
Good point! Opened #268 for it.